tests/java/test-app/pom.xml

```diff
@@ -133,79 +128,79 @@
           </dependency>
         </dependencies>
       </plugin>
-      <plugin>
-        <artifactId>maven-resources-plugin</artifactId>
-        <version>3.1.0</version>
-        <executions>
-          <execution>
-            <id>copy-resources</id>
-            <phase>package</phase>
-            <goals>
-              <goal>copy-resources</goal>
-            </goals>
-            <configuration>
-              <outputDirectory>${project.build.directory}/${docker.image.src.root}</outputDirectory>
-              <resources>
-                <!-- Copy entire Dockerfile image structure, except build -->
-                <resource>
-                  <directory>${docker.image.src.root}</directory>
-                  <exclude>**/build/**</exclude>
-                </resource>
-              </resources>
-            </configuration>
-          </execution>
-        </executions>
-      </plugin>
-      <plugin>
-        <groupId>com.spotify</groupId>
-        <artifactId>dockerfile-maven-plugin</artifactId>
-        <version>1.4.10</version>
-        <executions>
-          <execution>
-            <id>alpine</id>
-            <goals>
-              <goal>build</goal>
-              <goal>push</goal>
-            </goals>
-            <configuration>
-              <!-- <repository>${docker.registry}/${project.artifactId}/alpine</repository> -->
-              <dockerfile>${project.build.directory}/${docker.image.src.root}/alpine/Dockerfile</dockerfile>
-            </configuration>
-          </execution>
-          <execution>
-            <id>debian</id>
-            <goals>
-              <goal>build</goal>
-              <goal>push</goal>
-            </goals>
-            <configuration>
-              <!-- <repository>${docker.registry}/${project.artifactId}/debian</repository> -->
-              <dockerfile>${project.build.directory}/${docker.image.src.root}/debian/Dockerfile</dockerfile>
-            </configuration>
-          </execution>
-          <execution>
-            <id>ubuntu</id>
-            <goals>
-              <goal>build</goal>
-              <goal>push</goal>
-            </goals>
-            <configuration>
-              <!-- <repository>${docker.registry}/${project.artifactId}/ubuntu</repository> -->
-              <dockerfile>${project.build.directory}/${docker.image.src.root}/ubuntu/Dockerfile</dockerfile>
-            </configuration>
-          </execution>
-        </executions>
-        <configuration>
-          <tag>${project.version}</tag>
-          <contextDirectory>${project.build.directory}</contextDirectory>
-          <dockerConfigFile>${docker.config.file}</dockerConfigFile>
-          <verbose>true</verbose>
-          <skipDockerInfo>true</skipDockerInfo>
-          <buildArgs>
-            <JAR_FILE>${project.build.finalName}.jar</JAR_FILE>
-          </buildArgs>
-        </configuration>
-      </plugin>
+<!--      <plugin>-->
```

Comment:
how about putting all this in a profile instead? that way we can still build these locally for running the testapp